CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ Sections are used under each version as follows:
 
 ### Fixed
 
+- Fixed [BUG] On Create Sibling or Child Note, link to child or sibling incorrect #24
+- Fixed [BUG] Weird space added when making link to zettle
+#30
```

Comment:
This is fine to me now. But I think we should use a tool to generate the changelog in the future. It's much easier.

Reply:
I agree but that's not yet in place